### ISSUE TITLE
During training the data , it just change class one accuracy,but the others still show zero

### ISSUE THREAD

**Issue by mciky** [tkuanlun350/Tensorflow-SegNet#39]
2018-04-23 12:37:04.018255: step 110, loss = 1.55 (7.1 examples/sec; 0.281 sec/batch)
accuracy = 0.999809
mean IU  = 0.076908
    class # 0 accuracy = 0.999809
    class # 1 accuracy = 0.000000
    class # 2 accuracy = 0.000000
    class # 3 accuracy = 0.000000
    class # 4 accuracy = 0.000000
    class # 5 accuracy = 0.000000
    class # 6 accuracy = 0.000000
    class # 7 accuracy = 0.000000
    class # 8 accuracy = 0.000000
    class # 9 accuracy = 0.000000
    class # 10 accuracy = 0.000000
    class # 11 accuracy = 0.000000
    class # 12 accuracy = 0.000000
    class # 13 accuracy = 0.000000
    class # 14 accuracy = 0.000000
    class # 15 accuracy = 0.000000
    class # 16 accuracy = 0.000000
    class # 17 accuracy = 0.000000
    class # 18 accuracy = 0.000000
    class # 19 accuracy = 0.000000
    class # 20 accuracy = 0.000000
    class # 21 accuracy = 0.000000
    class # 22 accuracy = 0.000000
    class # 23 accuracy = 0.000000
    class # 24 accuracy = 0.000000
    class # 25 accuracy = 0.000000
    class # 26 accuracy = 0.000000
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
2018-04-23 12:37:07.865742: step 120, loss = 1.54 (7.1 examples/sec; 0.281 sec/batch)
accuracy = 0.999881
mean IU  = 0.076914
    class # 0 accuracy = 0.999881
    class # 1 accuracy = 0.000000
    class # 2 accuracy = 0.000000
    class # 3 accuracy = 0.000000
    class # 4 accuracy = 0.000000
    class # 5 accuracy = 0.000000
    class # 6 accuracy = 0.000000
    class # 7 accuracy = 0.000000
    class # 8 accuracy = 0.000000
    class # 9 accuracy = 0.000000
    class # 10 accuracy = 0.000000
    class # 11 accuracy = 0.000000
    class # 12 accuracy = 0.000000
    class # 13 accuracy = 0.000000
    class # 14 accuracy = 0.000000
    class # 15 accuracy = 0.000000
    class # 16 accuracy = 0.000000
    class # 17 accuracy = 0.000000
    class # 18 accuracy = 0.000000
    class # 19 accuracy = 0.000000
    class # 20 accuracy = 0.000000
    class # 21 accuracy = 0.000000
    class # 22 accuracy = 0.000000
    class # 23 accuracy = 0.000000
    class # 24 accuracy = 0.000000
    class # 25 accuracy = 0.000000
    class # 26 accuracy = 0.000000
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
2018-04-23 12:37:11.694385: step 130, loss = 1.53 (7.3 examples/sec; 0.273 sec/batch)
accuracy = 1.000000
mean IU  = 1.000000
    class # 0 accuracy = 1.000000
    class # 1 accuracy = 0.000000
    class # 2 accuracy = 0.000000
    class # 3 accuracy = 0.000000
    class # 4 accuracy = 0.000000
    class # 5 accuracy = 0.000000
    class # 6 accuracy = 0.000000
    class # 7 accuracy = 0.000000
    class # 8 accuracy = 0.000000
    class # 9 accuracy = 0.000000
    class # 10 accuracy = 0.000000
    class # 11 accuracy = 0.000000
    class # 12 accuracy = 0.000000
    class # 13 accuracy = 0.000000
    class # 14 accuracy = 0.000000
    class # 15 accuracy = 0.000000
    class # 16 accuracy = 0.000000
    class # 17 accuracy = 0.000000
    class # 18 accuracy = 0.000000
    class # 19 accuracy = 0.000000
    class # 20 accuracy = 0.000000
    class # 21 accuracy = 0.000000
    class # 22 accuracy = 0.000000
    class # 23 accuracy = 0.000000
    class # 24 accuracy = 0.000000
    class # 25 accuracy = 0.000000
    class # 26 accuracy = 0.000000
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
2018-04-23 12:37:15.534847: step 140, loss = 1.52 (7.4 examples/sec; 0.272 sec/batch)
accuracy = 1.000000
mean IU  = 1.000000
    class # 0 accuracy = 1.000000
    class # 1 accuracy = 0.000000
    class # 2 accuracy = 0.000000
    class # 3 accuracy = 0.000000
    class # 4 accuracy = 0.000000
    class # 5 accuracy = 0.000000
    class # 6 accuracy = 0.000000
    class # 7 accuracy = 0.000000
    class # 8 accuracy = 0.000000
    class # 9 accuracy = 0.000000
    class # 10 accuracy = 0.000000
    class # 11 accuracy = 0.000000
    class # 12 accuracy = 0.000000
    class # 13 accuracy = 0.000000
    class # 14 accuracy = 0.000000
    class # 15 accuracy = 0.000000
    class # 16 accuracy = 0.000000
    class # 17 accuracy = 0.000000
    class # 18 accuracy = 0.000000
    class # 19 accuracy = 0.000000
    class # 20 accuracy = 0.000000
    class # 21 accuracy = 0.000000
    class # 22 accuracy = 0.000000
    class # 23 accuracy = 0.000000
    class # 24 accuracy = 0.000000
    class # 25 accuracy = 0.000000
    class # 26 accuracy = 0.000000
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
[  2 360 480   3] [  2 360 480   1]
2018-04-23 12:37:19.353283: step 150, loss = 1.51 (7.4 examples/sec; 0.272 sec/batch)
accuracy = 0.999844
